server/notebooks/views.py

```diff
@@ -8,6 +9,8 @@
 from ..base.models import User
 from ..views import get_user_info_dict
 
+env = environ.Env()
```

Comment:
more common practice would be to define some variables in the settings file (`server/settings.py`) which read from the environment and then imported them here.

Then do something like:

```py
from ..settings import (APP_VERSION_STRING,
                        EVAL_FRAME_ORIGIN)
```

Reply:
ah that makes total sense, thanks!